### DISCUSSION TITLE
Interacting with point labels

### DISCUSSION THREAD

**Discussion by machur** [pyvista/pyvista#4379] [Q&A]
Hi, I'm using plotter's "add_point_labels" method to display descriptions for some meshes. My goal is to make the "black box" clickable, so that users may click on it to hide it. Is there an existing mechanism to add such handlers? There is a "pickable" flag in "add_point_labels" method, but it's only used for the points though. Would you please advise?

![image](https://user-images.githubusercontent.com/4650515/236010220-c8c42f8e-65d8-49f8-85c4-f4c57466459b.png)

**Comment by danelinquist** [comment]
Hi Machur,

I ran into a similar problem with a project I'm working on. While I did not find an existing mechanism to select the label directly, I did find a workaround. I took some time to document this on my discussion post here: [https://github.com/pyvista/pyvista/discussions/3976]. 
This should work for your use case as well by modifying the callback routine to hide the label when clicked.
